src/docs/product/issues/issue-details/index.mdx

```diff
@@ -19,6 +19,7 @@ There are two categories of issues: [error issues](/product/issues/issue-details
 - **Trace Navigator** - An abbreviated view of the related [trace](/product/sentry-basics/tracing/distributed-tracing) for the current transaction.
 - **Suspect Commits** - A commit that's been identified as potentially having caused an error event.
 - **Tags** - Searchable key/value string pairs providing information such as the browser or device.
+- **Screenshot** - Screenshot taken when a user experiences an error, only available on certain SDKs.
```

Comment:
Because I've added the list on the error issues page, I'm just linking to that from here instead of doing a platform redirect.
```suggestion
- **Screenshot** - Screenshot taken when a user experiences an error; only available on [certain SDKs](/product/issues/issue-details/error-issues/#screenshots).
```